package/kedro_viz/api/rest/responses/save_responses.py

```diff
@@ -0,0 +1,99 @@
+"""`kedro_viz.api.rest.responses.save_responses` contains response classes
+and utility functions for the `/deploy` REST endpoint"""
```

Comment:
this is more than just `/deploy` right ? this is also used in kedro viz --save-file I think

Reply:
I will update the text